### DISCUSSION TITLE
How to sintall this on linux, please

### DISCUSSION THREAD

**Discussion by renben-dev** [dacap/clip#56] [Q&A]
Hello, I'm a complete noob to github etc...,
I have cmake installed. How to I install clip libraty in my linux?? many thanks

**Comment by dacap** [comment]
HI @renzolinux, actually the `clip` library is created in a way to be used as a static library (not installed). You can check the ["How to use it"](https://github.com/dacap/clip/wiki#how-to-use-it) section in the wiki.
